### ISSUE TITLE
(Multi)tile verplaatsen naar ander dashboard

### ISSUE THREAD

**Issue by ronaldsomers** [mennovanhout/flutter-hdashboards-issues#138]
Mogelijkheid tot het verplaatsen of kopiëren/plakken van (multi)tile(s) naar ander dashboard.

**Comment by ronaldsomers** [comment]
Nu ik bezig ben met HDcompanion zie ik dat de identifier nrs van elke tegel op een ander dashboard afwijkt. 🧐

**Comment by mennovanhout** [comment]
Yes, dit is een beetje een interesant vraagstuk. Sommige mensen willen de onafhankelijkheid houden andere weer niet. Ik ga sowieso met je meerdere identifier vraagstuk aan de gang.

**Comment by ronaldsomers** [comment]
> Yes, dit is een beetje een interesant vraagstuk. Sommige mensen willen de onafhankelijkheid houden andere weer niet. Ik ga sowieso met je meerdere identifier vraagstuk aan de gang.

Wellicht de vraag bij toevoegen device wat je wil.
